package.json

```diff
@@ -19,15 +19,13 @@
     "link:emails:js": "rm -rf emails; ln -s packages/cli/src/generator_templates/js/emails emails;",
     "build": "preconstruct build",
     "test": "yarn link:emails; yarn jest",
+    "test:integration": "./scripts/assert-free-ports.sh; MAILING_CI=true MAILING_DATABASE_URL=$MAILING_DATABASE_URL_TEST WEB_DATABASE_URL=$WEB_DATABASE_URL_TEST start-test ci:cli :3883 ci:web :3000 \"yarn link:emails; yarn jest -c jest.integration.config.ts $TEST_FLAGS\"",
+    "test:integration:watch": "./scripts/assert-free-ports.sh; MAILING_CI=true MAILING_DATABASE_URL=$MAILING_DATABASE_URL_TEST WEB_DATABASE_URL=$WEB_DATABASE_URL_TEST start-test ci:cli :3883 ci:web :3000 \"yarn link:emails; yarn jest -c jest.integration.config.ts --watch\"",
```

Comment:
@psugihara thought we needed --runInBand here for the tests to run serially (in addition to the related stuff in the jest config).  is that not so?

Reply:
setting maxWorkers to 1 in jest.integration.config.ts has the same effect as setting runInBand 👍

Reply:
Yeah, when we had tried before it didn't seem to apply correctly to `--watch`. But this seems to be working as expected for me 👍